inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/hswish.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2019 Intel Corporation
```

Comment:
Why `hswish` single layer test did not created/instantiated?

Reply:
HSwish is not yet in the opset. So this test covers only plugin optimization

Reply:
HSwish has been added in the opset. So there is now a single layer test for hswish